server/index.js

```diff
@@ -5,12 +5,16 @@ const bodyparser = require('body-parser');
 const db = require('../database/index.js')
 const port = 3001;
 
-app.use(express.static(__dirname + '../public/'));
+app.use(express.static(__dirname + '/../public/'));
```

Comment:
My airbnb linter asked for "app.use(express.static(path.join(__dirname, '../public')));" in case this line throws yours for a loop.